apps/web/lib/reschedule/[uid]/getServerSideProps.ts

```diff
@@ -116,7 +117,8 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   if (
     isDisabledRescheduling ||
     (!allowRescheduleForCancelledBooking &&
-      (booking.status === BookingStatus.CANCELLED || booking.status === BookingStatus.REJECTED))
+      (booking.status === BookingStatus.CANCELLED || booking.status === BookingStatus.REJECTED) &&
+      booking.eventType?.disableReschedulingCancelledBookings)
```

Comment:
I think we could allow rescheduling if `allowRescheduleForCancelledBooking` query param is set(regardless of the value of `eventType?.disableReschedulingCancelledBooking`)  as it passed via query param. I don't remember why this feature was implemented but it was needed.